contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,104 @@
+---
+date: 2023-02-19
+title: Becoming a 10x engineer - why user interviews are key
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - luke-harries
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product engineers
+---
+
+When it comes to becoming a 10x engineer, most people think about writing more code, buying more monitors, or using the latest framework where everything is a macro. But, in reality, leverage comes from choosing the right problem, building the right solution, and using the right tools. This can only happen when you have a deep understanding of your users. It's the only way to build something that people want.
```

Comment:
```suggestion
When it comes to becoming a 10x engineer, most people think about improving their typing speed, buying more monitors, or using the latest framework where everything is a macro. But, in reality, leverage comes from choosing the right problem, building the right solution, and using the right tools. This can only happen when you have a deep understanding of your users. It's the only way to build something that people want.
```